### PR TITLE
branch testing

### PR DESCRIPTION
**Pull request by NarayanaReddy7**
* feature request for java 
* Bug Fix for Java

### CODE REVIEW

File: second.java [hunk 1]
```diff
@@ -0,0 +1 @@
+this is the second pr for testing featire branch
```
Comment (NarayanaReddy7) [inline second.java:1]: This code is not right. Please make changes.

**Comment by samrin1234** [comment]
The code is right

File: second.java (hunk 1 above)
Comment (NarayanaReddy7) [reply second.java:1]: Made changed to the code. Please review

Comment (samrin1234) [reply second.java:1]: okay i will update

**Comment by yashwanthnallapeta** [comment]
Go head

**Pull request merged**